nocache/search.go

```diff
@@ -67,7 +67,7 @@ func (ft *FullText) search(sp SearchParams) []map[string]any {
 
 	// Variables for storing the smallest words array
 	var (
-		smallest      int = 0
+		smallest      int = 0 //nolint:ineffassign
```

Comment:
I ignore these, as their inoffensive, and it imo looks a bit weird to just declare the type. Maybe I'm not enough of a hardcore gopher yet 🤓